x-pack/plugins/infra/public/containers/map/with_map.ts

```diff
@@ -3,50 +3,41 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import moment from 'moment';
 import { graphql } from 'react-apollo';
-import { InfraMetricType, InfraPathType, MapQuery } from '../../../common/graphql/types';
-import { InfraWaffleMapGroup } from '../../lib/lib';
+import { MapQuery } from '../../../common/graphql/types';
+import { InfraOptions, InfraWaffleMapGroup } from '../../lib/lib';
 import { nodesToWaffleMap } from './nodes_to_wafflemap';
 import { mapQuery } from './query';
 
-interface ChildProps {
+interface IncomingProps {
+  options: InfraOptions;
```

Comment:
IHMO if we passed in the options as separate props they could be pulled from different containers. That would improve reuse and make incremental refactoring easier.

Reply:
Do you mean instead of passing a global options object we just pass in the individual names like `path`, `sourceId`, `timerange` etc?

Reply:
Exactly. That would allow us to move them into the redux store incrementally without much disruption. For now the `IncomingProps` could just be the `InfraOptions` themselves.

Reply:
Ok